src/components/Chat/Chat.tsx

```diff
@@ -794,6 +794,12 @@ function ChatLine(props) {{{
         );
       }
 
+      if (body.substr(0, 9) === '/senseis ') {
```

Comment:
I tend to avoid regexp because I am not too familiar with it, but this is a much cleaner solution, It'll be good for me.